lib/internal/Magento/Framework/Data/Form/Element/Editor.php

```diff
@@ -286,15 +286,19 @@ protected function _getPluginButtonsHtml($visible = true)
 
         // Button to media images insertion window
         if ($this->getConfig('add_images')) {
+            $mediaGallery = (null !== $this->getConfig('is_new_media_gallery')) ?
+                          $this->getConfig('is_new_media_gallery') : 0;
```

Comment:
Could be better to typecast the value of config `(int)$this->getConfig('is_new_media_gallery')`

Also null coalescing operator could look more readable
`$mediaGallery = (int)$this->getConfig('is_new_media_gallery') : 0;`